client/sysinfo/src/lib.rs

```diff
@@ -44,13 +44,13 @@ pub const TARGET_ENV: &str = include_str!(concat!(env!("OUT_DIR"), "/target_env.
 #[derive(Clone, Debug, serde::Serialize)]
 pub struct HwBench {
 	/// The CPU speed, as measured in how many MB/s it can hash using the BLAKE2b-256 hash.
-	pub cpu_hashrate_score: u64,
+	pub cpu_hashrate_score: Throughput,
 	/// Memory bandwidth in MB/s, calculated by measuring the throughput of `memcpy`.
-	pub memory_memcpy_score: u64,
+	pub memory_memcpy_score: Throughput,
 	/// Sequential disk write speed in MB/s.
-	pub disk_sequential_write_score: Option<u64>,
+	pub disk_sequential_write_score: Option<Throughput>,
 	/// Random disk write speed in MB/s.
-	pub disk_random_write_score: Option<u64>,
+	pub disk_random_write_score: Option<Throughput>,
 }
```

Comment:
Could you please also add a test which serializes this structure into JSON and compares it to a preset string? (Since the format needs to explicitly match what telemetry expects we should have a test that enforces it.)

Reply:
https://github.com/paritytech/substrate/blob/952030cfa6f11be6aef938e5359064c4cf6b30a9/utils/frame/benchmarking-cli/src/machine/hardware.rs#L34
This didn't work without the custom deserialization.